test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -0,0 +1,241 @@
+#include "test/common/upstream/load_balancer_fuzz_base.h"
+
+#include "test/common/upstream/utility.h"
+
+#define MAX_NUM_HOSTS_IN_PRIORITY_LEVEL 256
```

Comment:
In Envoy, prever `constexpr uint64 MaxHostInPriorityLevel` in anonymous namespace.

Reply:
Changed.